polyfill/lib/timezone.mjs

```diff
@@ -69,23 +69,16 @@ export class TimeZone {
       }
     }
 
-    const utcns = ES.GetEpochFromParts(
-      year,
-      month,
-      day,
-      hour,
-      minute,
-      second,
-      millisecond,
-      microsecond,
-      nanosecond
-    );
+    const utcns = ES.GetEpochFromParts(year, month, day, hour, minute, second, millisecond, microsecond, nanosecond);
     if (utcns === null) throw new RangeError('DateTime outside of supported range');
     const before = ES.GetTimeZoneOffsetNanoseconds(utcns.minus(bigInt(86400 * 1e9)), GetSlot(this, IDENTIFIER));
     const after = ES.GetTimeZoneOffsetNanoseconds(utcns.plus(bigInt(86400 * 1e9)), GetSlot(this, IDENTIFIER));
-    const diff = ES.ToDuration({
-      nanoseconds: after.minus(before)
-    }, 'reject');
+    const diff = ES.ToDuration(
+      {
+        nanoseconds: after.minus(before)
+      },
+      'reject'
+    );
```

Comment:
```suggestion
    const nanoseconds = after.minus(before);
    const diff = ES.ToDuration({ nanoseconds }, 'reject');
```

Reply:
admit opinion is more political than technical (and unlikely to change minds),
but prefer switching enforcement from single-quote to double-quote:

#### pros:
1. better interoperability with sql-database having queries with single-quotes, e.g.:
```js
sqljsWorker.postMessage({
    action: "exec",
    sql: "INSERT INTO myTable (username, user_level) VALUES ('johndoe', 'admin');"
});
```

2. easier to inline multi-line javascript-code in [linux] shell-scripts like this real-world example:
```bash
#!/bin/sh
shImageToDataUri () {
# this shell-function will convert the image $1 to a data-uri string
    node -e '
/* jslint utility2:true */
(function () {
"use strict";
console.log(
    "data:image/"
    + require("path").extname(process.argv[1]).slice(1)
    + ";base64,"
    + require("fs").readFileSync(process.argv[1]).toString("base64")
);
}());
' "$1"
}

shImageToDataUri favicon.png
# stdout
# data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwBAMAAAClLOS0AA...
```

3. jslint forbids single-quote strings (yea, this ones political but cant help myself ;)



#### cons:
1. disliked by new-users coming from python-background (which i once was)

2. awkward escaping of double-quotes when manipulating serialized JSON data, e.g.
```js
constantData = "{\"foo\":\"bar\"}";
```

Reply:
Single quotes for JS strings is by an exceedingly large margin the prevailing common style in the community.

SQL queries are a vanishingly small use case for JS compared to literally everything else that uses strings, inlining JS code like that is a bad practice anyways, and in any case you can use backticks if you want to avoid escaping.

Reply:
sqlite-queries (with isostrings) is the future of state-management in browsers.  the indexeddb/mongodb experiments to do table-joins in userland-javascript have ended in failure.

Reply:
The eslint rule proposed in this PR is for single quotes with an exception to allow double quotes when the string contains single quotes. Prettier does the same thing by default. So you can still quote SQL queries with double quotes, not that we would use any in this polyfill...